isis/src/base/objs/EquatorialCylindricalShape/EquatorialCylindricalShape.cpp

```diff
@@ -407,9 +410,24 @@ namespace Isis {
 
     }
 
+    // Do nothing since the DEM intersection was already successful
     setHasIntersection(true);
     return hasIntersection();
   }
-  // Do nothing since the DEM intersection was already successful
-}
 
+
+  /**
+   * Override of virtual function for intersectEllipsoid
+   *
+   * param &a SpiceDouble reference to be set
+   *
```

Comment:
remove lines between params and give more descriptive documentation. (But this is a comment just for future reference, since I think you should remove these parameters - see other comments)